versions/3.0.md

```diff
@@ -763,7 +763,7 @@ Describes a single API operation on a path.
 Field Name | Type | Description
 ---|:---:|---
 <a name="operationTags"></a>tags | [`string`] | A list of tags for API documentation control. Tags can be used for logical grouping of operations by resources or any other qualifier.
-<a name="operationSummary"></a>summary | `string` | A short summary of what the operation does. For maximum readability in editing or documentation generation tools, this field SHOULD be less than 120 characters.
```

Comment:
I'm OK with not limiting this to 120, but I think we should mention that many tools (including Swagger UI, swagger-codegen HTML output) render the summary strings as *titles* and long summaries result in a degraded experience. (Ditto for title in schemas/models) So some context explaining *why* one should keep summary *short* (a single line) would be helpful.